tensorflow/lite/micro/examples/person_detection/spresense/src/spresense_image_provider.cc

```diff
@@ -0,0 +1,34 @@
+/* Copyright 2021 The TensorFlow Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+==============================================================================*/
+
+// The SPRESENSE_CONFIG_H is defined on compiler option.
+// It contains "nuttx/config.h" from Spresense SDK to see the configurated parameters.
+#include SPRESENSE_CONFIG_H
+#include "spresense_image_provider.h"
+
+#include "tensorflow/lite/micro/examples/person_detection/image_provider.h"
+#include "tensorflow/lite/micro/examples/person_detection/model_settings.h"
+
+TfLiteStatus GetImage(tflite::ErrorReporter* error_reporter, int image_width,
+                      int image_height, int channels, int8_t* image_data) {
+  if (spresense_getimage((unsigned char *)image_data)==0)
```

Comment:
I would expect this brace style to fail the style check, as an example.

Reply:
Thank you for your review, @petewarden 
I will try test_code_style.sh.

I have one question about your comment.
The function name of "spresense_getimage()" is provided from spresense platform.
How should I do for following your style in such a case?

Options I think I can take :
  1. make a header file to rename it like "#define GetImageFromSpresense spresense_getimage".
  2. Change the function name in spresense platform.
 
2nd option is  simple and it can be taken in this case because the function is made for this.
So I will take this.

In general, I just wondering when you want to use a function from outside and if it is not fit on your style, how do you use it?
Could you let me know it?

Reply:
Hi @petewarden 
I tried test_code_style.sh but I got errors as below.

Is this change request from the result of test_code_style.sh execution?
I will change it this only soon.

And, I think I should write this question on an issue but is this error known issue?

==== Error Log ====
$ ./test_code_style.sh 
~/work/tmp/tensorflow/tensorflow/lite ~/work/tmp/tensorflow
  File "micro/tools/make/downloads/pigweed/pw_presubmit/py/pw_presubmit/pigweed_presubmit.py", line 55
    f'{ctx.root}#:python.install',
                                ^
SyntaxError: invalid syntax
  File "micro/tools/make/downloads/pigweed/pw_presubmit/py/pw_presubmit/format_code.py", line 46
    _LOG: logging.Logger = logging.getLogger(__name__)
        ^
SyntaxError: invalid syntax
~/work/tmp/tensorflow